config.go

```diff
@@ -154,12 +155,33 @@ func ConfigStringToClusterConfig(configString string) (*gocql.ClusterConfig, err
 				case "password":
 					passwordAuthenticator.Password = value
 					clusterConfig.Authenticator = passwordAuthenticator
+				case "enableHostVerification":
+					if sslOptions == nil {
+						sslOptions = &gocql.SslOptions{}
+					}
+					sslOptions.EnableHostVerification = (value == "true")
+				case "certPath":
+					if sslOptions == nil {
+						sslOptions = &gocql.SslOptions{}
+					}
+					sslOptions.CertPath = value
```

Comment:
For this one and the next two, was just wondering is there any special checks that should be done? If not, no worries. Just was curious.

Reply:
i think it's better for this function to just convert the `string` to `*gocql.ClusterConfig` and leave any further validation elsewhere